wifi-qr

```diff
@@ -66,15 +66,17 @@ main_menu() {
   elif [[ "$CHOICE" =~ "See version" ]]; then
     zenity --info --icon-name=wifi-qr --title="WiFi QR Version" --no-wrap --text=" This is wifi-qr version $VERSION"
     main_menu
- else
+  else
 	exit 0
   fi
 }
 
 
 #GUI
 call_wifi_gui() {
-   IFS=$'\r\n' GLOBIGNORE='*' command eval  'wifilist=($(nmcli -g NAME connection))'
+  check_for_package
+  nmcli radio wifi on
```

Comment:
In case user is using the script without turning wifi on, the script will not return error about not wifi is in off stage. That’s why....There might be better alternatives

Reply:
Sometime we use without wifi
WiFi-QR is support Scan from other devices or Save Network Credential to QR with other.
its no need wifi or connection.

Reply:
I see now ...